src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -112,6 +112,17 @@
                 <MenuItem Name="help_cm"
                           Header="{x:Static p:Resources.NodeContextMenuHelp}"
                           Command="{Binding Path=ShowHelpCommand}" />
+                <MenuItem Header="{x:Static p:Resources.ContextSelectConnectedHeader}">
```

Comment:
So this one should go above the `help_cm` as per Sols comment

Reply:
ok let me fix that, and then it should be ready for Autodesk, @SHKnudsen, should I also update this branch from master before creating the PR?

Reply:
> should I also update this branch from master before creating the PR
Yep

Also lets make sure to resolve the comments on the tests before we submit, at least these https://github.com/martinstacey/Dynamo/pull/24#discussion_r651559691